packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -109,6 +112,22 @@ class Chat extends Widget<Properties> {
     this._messageBox = this._createComponent($messageBox, MessageBox, configuration);
   }
 
+  _updateRootAria(): void {
+    const aria = {
+      role: 'group',
+      label: 'Chat',
+    };
+
+    this.setAria(aria, this.$element());
+  }
+
+  _updateMessageBoxAria(): void {
+    const $emptyView = this._messageList.$element().find(`.${CHAT_MESSAGELIST_EMPTY_VIEW_CLASS}`);
+    const emptyViewId = $emptyView?.attr('id') ?? null;
```

Comment:
Can we implement a public method inside the messageList component that will return the emptyViewId value if the empty view is rendered, and null in other cases? Seems, in this case we can avoid look for emptyview element on Chat's level. Something like this:
```
this._messageList.getEmptyViewId(): id | null {
    return this. _isEmpty() ? $(this.element()).find(`.${CHAT_MESSAGELIST_EMPTY_VIEW_CLASS}`).attr('id') ?? null;
}
```

Reply:
[ddd98c2f8675ba0e9ea72a8e3cde9824787a94a4]